bentoml/deployment/serverless.py

```diff
@@ -92,3 +98,28 @@ def deploy_with_serverless(platform, archive_path, extra_args):
     output_path = generate_serverless_bundle(bento_service, platform, archive_path, extra_args)
     subprocess.call(['serverless', 'deploy'], cwd=output_path)
     return output_path
+
+
+def delete_serverless_deployment(platform, service_name, additional_info):
+    yaml = YAML()
+    temp_dir = tempfile.mkdtemp()
```

Comment:
try `with TempDirectory as tempdir:` in `utils/tempdir.py`? the main difference is that it adds a prefix indicating the directory is related to bentoml package and it cleans up the directory afterward